.github/workflows/checks.yml

```diff
@@ -98,14 +94,91 @@ jobs:
         run: |
           poetry install
 
+      # ====================== CONTRACTS v2_6 ====================== #
+      - name: Cache contracts v2_6
+        id: cache-contracts-v2_6
+        uses: actions/cache@v3
+        with:
+          path: starknet_py/tests/e2e/mock/contracts_compiled_v2_6
+          key: ${{ runner.os }}-contracts-${{ hashFiles('starknet_py/tests/e2e/mock/contracts_v2_6') }}
+    
+      - uses: actions/checkout@v4
+        with:
+          repository: starkware-libs/cairo
```

Comment:
why not use scarb?

Reply:
These changes are moved to the new PR https://github.com/software-mansion/starknet.py/pull/1322